web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,37 +1,44 @@
 import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+/** Anchored metadata AKA "Additional Info" - Editable only with a signed Ledger TX. */
 export interface ProjectMetadataLD {
   '@context': Context;
-  '@type': string;
+  '@type': string; // regen:C01-Project
   'schema:name': string;
-  'schema:image': UrlType;
-  'schema:creditText': string;
-  'schema:description'?: string;
   'schema:location': any;
   'regen:projectType': string;
   'regen:projectActivity': ProjectActivity;
-  'regen:projectDeveloper': ProjectStakeholder;
-  'regen:landSteward': ProjectStakeholder;
-  'regen:landOwner': ProjectStakeholder;
-  'regen:projectOriginator': ProjectStakeholder;
   'regen:offsetGenerationMethod': string;
   // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
   'http://regen.network/offsetGenerationMethod'?: string;
   'regen:projectSize': ProjectSize;
   'regen:projectStartDate': TypeValue;
   'regen:projectEndDate': TypeValue;
-  'regen:boundaries': TypeValue;
-  'regen:creditClass': CreditClass;
-  'regen:landManagementActions': LandManagementActions;
+}
+
+/** Un-anchored metadata from our DB. This is editable without a Ledger TX. */
+export interface ProjectPageMetadataLD {
+  '@context': Context;
+  '@type': string; // regen:C01-Project-Page
+  'schema:image': UrlType;
+  'schema:creditText': string;
   'regen:galleryPhotos': UrlList;
   'regen:previewPhoto': UrlType;
   'regen:videoURL': UrlType;
+  'regen:projectDeveloper': ProjectStakeholder;
   'regen:glanceText': any;
+  'regen:landSteward': ProjectStakeholder;
+  'regen:landOwner': ProjectStakeholder;
+  'regen:projectOriginator': ProjectStakeholder;
   'regen:landStory': string;
   'regen:landStewardStory': string;
   'regen:landStewardPhoto': UrlType;
   'regen:projectQuote': ProjectQuote;
   'regen:landStewardStoryTitle': string;
+  'schema:description'?: string;
+  'regen:boundaries': TypeValue;
+  'regen:creditClass': CreditClass;
+  'regen:landManagementActions': LandManagementActions;
 }
 
 interface Context {
```

Comment:
nit/tangent/optional: we could technically be more strict with the `Context` interface:

```
interface Context {
  schema: 'http://schema.org/';
  regen: 'http://regen.network/';
  qudt: 'http://qudt.org/schema/qudt/';
  unit: 'http://qudt.org/vocab/unit/';
  xsd: 'http://www.w3.org/2001/XMLSchema#';
}
```

Reply:
but there are some fields in the context that may need to be marked as optional. i.e. not all metadata needs the `xsd` schema for example.